server.js

```diff
@@ -8,11 +8,9 @@ module.exports = app
 // Set the template engine to use nunjucks
 app.set('view engine', 'nunjucks')
 
-// Set the location of the component and template files
+// Set the location of the component files
 var appViews = [
-  path.join(__dirname, 'views'),
-  path.join(__dirname, '/node_modules/govuk_frontend_alpha/components/'),
-  path.join(__dirname, '/node_modules/govuk_frontend_alpha/templates/')
```

Comment:
If this line is removed, then the app is unable to find the template file in index.njk, so when I use: `{% extends "../node_modules/govuk_frontend_alpha/templates/govuk_template.njk" %}` I get a "template not found" error.